### PR TITLE
Release steps explicit documented

### PR DESCRIPTION
**Pull request by matzew**
showing exact release commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,11 +57,16 @@ and start the latest build, locally, like `docker run -p 18081:8080 -it aerogear
...
 ## Releasing the UnifiedPush Server
 
-The content of the [Release Process](https://github.com/aerogear/collateral/wiki/Release-Process-(Java)) is valid for this project as well. However, to build the `distribution` bundle, you need to include these profiles:
+The content of the [Release Process](https://github.com/aerogear/collateral/wiki/Release-Process-(Java)) is valid for this project as well. However, to build the full `distribution` bundle, you need to fire off the release like:
 
 ```
-mvn release:GOAL -Pdist,test
+## prepare the release and define the TAG and adjust the versions:
+mvn release:prepare -Dtag=x.y.z.Final -Darguments=-Dgpg.passphrase=$MY_SECRET_PASS_PHRASE -Pdist,test
+
```
Comment (pb82) [inline README.md]: The tag needs to be deleted and recreated from the previous commit after that step right?

File: README.md [hunk 2]
```diff
@@ -57,11 +57,16 @@ and start the latest build, locally, like `docker run -p 18081:8080 -it aerogear
...
+## run the actual release process and load the artifacts to JBoss Nexus
+mvn release:perform -DperformRelease=true -Darguments=-Dgpg.passphrase=$MY_SECRET_PASS_PHRASE -Dgpg.useagent=true -Pdist,test
```
Comment (pb82) [inline README.md]: The changes need to be pushed to master after this step.

Comment (wtrocki) [reply README.md]: Who will be allowed to do it? $MY_SECRET_PASS_PHRASE - is that something we should define before?

File: README.md (hunk 1 above)
Comment (matzew) [reply README.md]: yes, that's mentioned in the real release process document, the sentence before this snippet does actually link to it

File: README.md (hunk 2 above)
Comment (matzew) [reply README.md]: > The changes need to be pushed to master after this step.

same, the real document talks about that.  

This here is mainly to give the exact mvn commands

Comment (matzew) [reply README.md]: @wtrocki the release manager does it - see the release process for details

Comment (matzew) [reply README.md]: @wtrocki the link is here https://github.com/aerogear/collateral/wiki/Release-Process-(Java)

Comment (pb82) [reply README.md]: @wtrocki `MY_SECRET_PASS_PHRASE` refers to your gpg cert password.

**Pull request merged**